cloud/deployment/inspect/inspect.go

```diff
@@ -16,54 +16,54 @@ import (
+	ClusterID      string    `mapstructure:"cluster_id" yaml:"cluster_id" json:"cluster_id"`
+	ReleaseName    string    `mapstructure:"release_name" yaml:"release_name" json:"release_name"`
+	AirflowVersion string    `mapstructure:"airflow_version" yaml:"airflow_version" json:"airflow_version"`
+	Status         string    `mapstructure:"status" yaml:"status" json:"status"`
+	CreatedAt      time.Time `mapstructure:"created_at" yaml:"created_at" json:"created_at"`
+	UpdatedAt      time.Time `mapstructure:"updated_at" yaml:"updated_at" json:"updated_at"`
+	DeploymentURL  string    `mapstructure:"deployment_url" yaml:"deployment_url" json:"deployment_url"`
+	WebserverURL   string    `mapstructure:"webserver_url" yaml:"webserver_url" json:"webserver_url"`
 }
 
 type deploymentConfig struct {
-	Name           string `mapstructure:"name" yaml:"name" json:"name"`
-	Description    string `mapstructure:"description" yaml:"description" json:"description"`
-	RunTimeVersion string `mapstructure:"runtime_version" yaml:"runtime_version" json:"runtime_version"`
-	SchedulerAU    int    `mapstructure:"scheduler_au" yaml:"scheduler_au" json:"scheduler_au"`
-	SchedulerCount int    `mapstructure:"scheduler_count" yaml:"scheduler_count" json:"scheduler_count"`
-	ClusterID      string `mapstructure:"cluster_id" yaml:"cluster_id" json:"cluster_id"` // this is also in deploymentMetadata
+	Name             string `mapstructure:"name" yaml:"name" json:"name"`
+	Description      string `mapstructure:"description" yaml:"description" json:"description"`
+	RunTimeVersion   string `mapstructure:"runtime_version" yaml:"runtime_version" json:"runtime_version"`
+	DagDeployEnabled bool   `mapstructure:"dag_deploy_enabled" yaml:"dag_deploy_enabled" json:"dag_deploy_enabled"`
+	SchedulerAU      int    `mapstructure:"scheduler_au" yaml:"scheduler_au" json:"scheduler_au"`
+	SchedulerCount   int    `mapstructure:"scheduler_count" yaml:"scheduler_count" json:"scheduler_count"`
+	ClusterName      string `mapstructure:"cluster_name" yaml:"cluster_name" json:"cluster_name"`
+	WorkspaceName    string `mapstructure:"workspace_name" yaml:"workspace_name" json:"workspace_name"`
 }
 
-type workerq struct {
+type Workerq struct {
 	Name              string `mapstructure:"name" yaml:"name" json:"name"`
-	ID                string `mapstructure:"id" yaml:"id" json:"id"`
 	IsDefault         bool   `mapstructure:"is_default" yaml:"is_default" json:"is_default"`
 	MaxWorkerCount    int    `mapstructure:"max_worker_count" yaml:"max_worker_count" json:"max_worker_count"`
 	MinWorkerCount    int    `mapstructure:"min_worker_count" yaml:"min_worker_count" json:"min_worker_count"`
 	WorkerConcurrency int    `mapstructure:"worker_concurrency" yaml:"worker_concurrency" json:"worker_concurrency"`
-	NodePoolID        string `mapstructure:"node_pool_id" yaml:"node_pool_id" json:"node_pool_id"`
+	WorkerType        string `mapstructure:"worker_type" yaml:"worker_type" json:"worker_type"`
```

Comment:
Is there a reason we don't still use NodePoolID? This strongly assumes that we only have one node pool with a given worker type, which might not necessarily be the case in the future.

Reply:
Ah nvm, saw the conversion here:
`getWorkerTypeFromNodePoolID` Please disregard.